### ISSUE TITLE
DB Migration: Notifications: Phase 2: Check for read errors

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#5178]
Predecessors: #5177

Now that data is being read from postgres, we need to guarantee there are no regressions.

AC
- [ ] After #5177 has been deployed to prod for 1 week, sentry logs are checked for any errors that are linked to missing notifications.
- [ ] For any error found, a new issue is created that is linked to the sentry error

Estimate: 2 hours

**Comment by jordanh** [comment]
Keeping open
